internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriterOptions.java

```diff
@@ -69,6 +70,10 @@ public boolean isRmdChunkingEnabled() {
     return rmdChunkingEnabled;
   }
 
+  public Integer getMaxRecordSizeBytes() {
```

Comment:
nit only, let's resolve it in the upcoming nearline part of PR: we can just use int instead of Integer, same goes to the variable definition

Reply:
Changed to `int` and set default value in `Builder` rather than using `Integer` in the nearline changes.